### PR TITLE
chore: restructure backend, frontend to chatbot-core

### PR DESCRIPTION
**Pull request by lelouvincx**
## Description

Restructure into 2 projects: `chatbot-core` and `vanphongso`.

## Related Issue(s)

[If applicable, link to the issue(s) this PR addresses]
Ref #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,3 @@
...
-cp .env.example .env
-
-# Build the docker image
-docker-compose build
-
-# Run the docker container
-docker-compose up -d
-```
\ No newline at end of file
+boi boi boi
```
Comment (tuantran0910) [inline README.md:3]: What is this ?

**Review by tuantran0910 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lelouvincx) [reply README.md:3]: Fuck you
I am trying to sign my commit
Please ignore

Comment (greyyT) [reply README.md:3]: I think your behavior right now is not professional. Please test on your local github, not a pull request for everyone to see. Thank you. Good night.

cc @tuantran0910

Comment (tuantran0910) [reply README.md:3]: > Member

I do agree with you. We should punish him.

Comment (lelouvincx) [reply README.md:3]: No.
I need boi q to support us.

**Review by greyyT — APPROVED** [review]
LGTM

**Comment by lelouvincx** [comment]
Waiting for #27 to be done. Then this PR will be rebased from #27.

**Pull request merged**